sdkit/utils/file_utils.py

```diff
@@ -29,49 +35,39 @@ def save_images(images: list, dir_path: str, file_name='image', output_format='J
           and the returned value will be used as the actual file name. e.g `def fn(i): return 'foo' + i`
     * output_format: 'JPEG' or 'PNG'
     * output_quality: an integer between 0 and 100, used for JPEG
-    '''
-    if dir_path is None: return
+    """
+    if dir_path is None:
+        return
     os.makedirs(dir_path, exist_ok=True)
 
     for i, img in enumerate(images):
-        actual_file_name = file_name(i) if callable(file_name) else f'{file_name}_{i}'
+        actual_file_name = file_name(i) if callable(file_name) else f"{file_name}_{i}"
         path = os.path.join(dir_path, actual_file_name)
-        img.save(f'{path}.{output_format.lower()}', quality=output_quality)
+        img.save(f"{path}.{output_format.lower()}", quality=output_quality)
 
-def save_dicts(entries: list, dir_path: str, file_name='data', output_format='txt', file_format=''):
-    '''
+
+def save_dicts(entries: list, dir_path: str, file_name="data", output_format="txt"):
+    """
     * entries: a list of dictionaries
     * dir_path: the directory path where the files will be saved
     * file_name: the file name to save. Can be a string or a function.
         if a string, the actual file name will be `{file_name}_{index}`.
         if a function, the callback function will be passed the `index` (int),
           and the returned value will be used as the actual file name. e.g `def fn(i): return 'foo' + i`
-    * output_format: 'txt', 'json', or 'embed'
-        if 'embed', the metadata will be embedded in PNG files in tEXt chunks, and as EXIF UserComment for JPEG files
-    '''
-    if dir_path is None: return
+    * output_format: 'txt' or 'json'
+    """
+    if output_format.lower() not in ["json", "txt"]:
+        return
+    if dir_path is None:
+        return
     os.makedirs(dir_path, exist_ok=True)
 
     for i, metadata in enumerate(entries):
-        actual_file_name = file_name(i) if callable(file_name) else f'{file_name}_{i}'
+        actual_file_name = file_name(i) if callable(file_name) else f"{file_name}_{i}"
         path = os.path.join(dir_path, actual_file_name)
-
-        if output_format.lower() == 'embed' and file_format.lower() == 'png':
```

Comment:
Can you please rebase your branch with the latest changes in `main`? I think it's missing the recent changes for metadata embedding (and is therefore removing these changes). Thanks